packages/python-packages/doc-warden/README.md

```diff
@@ -179,12 +180,13 @@ known_content_issues:
   - ['sdk/template/azure-sdk-template/README.md','#1368']
   - ['sdk/template/azure-sdk-template/CHANGELOG.md','#1368']
 ```
-
 The above configuration tells `warden`...
 
 - The language within the repo is `java`
 - To ensure that a `README.md` is present at the root of the repository.
-- To omit any paths under `archive/` from the readme checks.
+- To omit **any** paths under `archive/` from the readme checks.
```

Comment:
Can you add an example of what the `**` means? I see you switched some things in https://github.com/Azure/azure-sdk-for-js/pull/7160. 

Also based on the docs at https://docs.python.org/2/library/fnmatch.html I'm not even sure what `**` does. It might be worth actually linking to those docs for folks to read more about the pattern matching.